packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -63,6 +68,10 @@ const propTypes = {
    * The first parameter is the event. The second parameter is the changed input value.
    */
   onChangeRaw: PropTypes.func,
+  /**
+   * A callback function triggered when the date input, hour input, or minute input receives focus. function(event)
```

Comment:
should we remove function(event)
```suggestion
   * A callback function triggered when the date input, hour input, or minute input receives focus.
```

Reply:
Updated: https://github.com/cerner/terra-framework/pull/588/commits/608599424d554dff2a4908132605c72032d51661